app/views/WhatYouOwe.scala.html

```diff
@@ -265,16 +265,32 @@
     </tr>
 }
 
+@getHeading = @{
+    if(isAgent) getPrefix("heading-agent") else getPrefix("heading")
+}
+
+@getPaymentsMadeBulletLinkText = @{
+    if(isAgent) messages("whatYouOwe.payments-made-bullet-agent-1.2") else messages("whatYouOwe.payments-made-bullet-1.2")
+}
+
+@getPaymentsMadeBulletText = @{
+    if(isAgent) messages("whatYouOwe.payments-made-bullet-agent-2") else messages("whatYouOwe.payments-made-bullet-2")
+}
+
+@getNoPaymentsDueText = @{
```

Comment:
worth to consider to create an extension method something like: messagesIfElse(Boolean, Key1, Key2) as this pattern appears very often across project

Reply:
story created => 
https://jira.tools.tax.service.gov.uk/browse/MISUV-4079

Reply:
sounds good, thanks